storybook/stories/chart/radarchart/RadarChart.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Radar, RadarChart, ResponsiveContainer } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Radar Chart',
+  component: RadarChart,
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <RadarChart data={args.data} width={600} height={500} outerRadius={150}>
```

Comment:
Can we make this even simpler, i.e. not even use the outerRadius? 
Further, if we do not pass width and height here, how does it size in a responsive container? 

Would this work? 
```suggestion
        <RadarChart data={args.data} >
```